docs/_blog/index.html

```diff
@@ -1,6 +1,6 @@
 ---
 layout: static-site-main
-title: Blog
+title: Blog (archive)
```

Comment:
Does `archive` here have a special meaning? Or do you just want to show that the blog here is archived?

Reply:
I meant that this blog only contains old articles and it is not the place to find the latest news about Scala 3 development. Does that make sense? Should I say it differently?

Reply:
I think this is fine, just wanted to make sure that the `archive` keyword in here didn't trigger some special behavior or something.